src/main/java/org/fundacion/jala/converter/core/ImageConverter.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Daniela Santa Cruz Andrade
+ */
+package org.fundacion.jala.converter.core;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.fundacion.jala.converter.core.exceptions.ConverterException;
+import org.fundacion.jala.converter.core.parameter.ImageParameter;
+import org.fundacion.jala.converter.core.results.Result;
+import java.io.IOException;
+
+/**
+ * This class converts a image to an specified format.
+ */
+public class ImageConverter {
+    private String startFirstCommand = "ffmpeg -i ";
+    private ImageParameter parameter;
+    private String format;
+    private String output;
+    private String pathOutput;
+    private String outputFileName;
+    private final int HIGH_LIMIT = 1000;
+    private static final Logger LOGGER = LogManager.getLogger();
+    private Result result;
+
+    public ImageConverter(final ImageParameter imageParameter) {
+        this.parameter = imageParameter;
+    }
+
+    /**
+     * Converts the input image.
+     *
+     * @throws ConverterException if process is interrupted.
+     */
+    public void convertImage() throws ConverterException {
+        String adaptPath = "\"" + parameter.getFilePath() + "\"";
+        format = parameter.getOutputFormat();
+        output = adaptPath.substring((adaptPath.lastIndexOf("\\") + 1), adaptPath.lastIndexOf(".") + 1) + format;
+        setOutputFileName(output);
+        pathOutput = adaptPath.substring(0, (adaptPath.lastIndexOf("archive"))) + "archive\\";
+        String ffmpegCommand = startFirstCommand + adaptPath + " ";
+        String parameters = changeSize() + grayScale() + "\" ";
+        String theCommand = ffmpegCommand + parameters + pathOutput + output  + "\" -y";
+        try {
+        Process process = Runtime.getRuntime().exec("cmd /c " + theCommand);
+        System.out.println(theCommand);
+        ThreadHandler errorHandler = new ThreadHandler(process.getErrorStream(), "Error Stream");
+        errorHandler.start();
+        ThreadHandler threadHandler = new ThreadHandler(process.getInputStream(), "Output Stream");
+        threadHandler.start();
+        LOGGER.info("start");
+            LOGGER.info("Execute Try");
+            process.waitFor();
+            LOGGER.info("finish");
+            System.out.println("exit code: " + process.exitValue());
+        } catch (InterruptedException | IOException exception) {
```

Comment:
Don't know if IOException should be here

Reply:
It is similar to other classes :)